notebooks/regression/tutorial_regression.md

```diff
@@ -234,7 +261,230 @@ All strategies except the Naive one give effective coverage close to the expecte
+    "cv_plus": Params(method="plus", cv=10),
+    "cv_minmax": Params(method="minmax", cv=10),
+    "jackknife_plus_ab": Params(method="plus", cv=Subsample(n_resamplings=50)),
+    "conformalized_quantile_regression": Params(method="quantile", cv="split", alpha=0.05)
+}
+y_pred, y_pis = {}, {}
+for strategy, params in STRATEGIES.items():
+    if strategy == "conformalized_quantile_regression":
+        mapie = MapieQuantileRegressor(polyn_model_quant, **params)
+        X_train, X_calib, y_train, y_calib = train_test_split(
+            X_train,
+            y_train,
+            test_size=0.5,
+            random_state=1
+        )
+        mapie.fit(X_train, y_train, X_calib, y_calib)
+        y_pred[strategy], y_pis[strategy] = mapie.predict(X_test)
+    else:  
+        mapie = MapieRegressor(polyn_model, **params)
+        mapie.fit(X_train, y_train)
+        y_pred[strategy], y_pis[strategy] = mapie.predict(X_test, alpha=0.05)
+```
+
+Let’s now compare the confidence intervals with the predicted intervals with obtained 
+by the Jackknife+, Jackknife-minmax, CV+, CV-minmax, Jackknife+-after-Boostrap, and quantile strategies. Note that for the Jackknife-after-Bootstrap method, we call the :class:`mapie.subsample.Subsample` object that allows us to train bootstrapped models.
+
+```python
+def plot_1d_data(
+    X_train,
+    y_train, 
+    X_test,
+    y_test,
+    y_sigma,
+    y_pred, 
+    y_pred_low, 
+    y_pred_up,
+    ax=None,
+    title=None
+):
+    ax.set_xlabel("x") ; ax.set_ylabel("y")
+    ax.fill_between(X_test, y_pred_low, y_pred_up, alpha=0.3)
+    ax.scatter(X_train, y_train, color="red", alpha=0.3, label="Training data")
+    ax.plot(X_test, y_test, color="gray", label="True confidence intervals")
+    ax.plot(X_test, y_test - y_sigma, color="gray", ls="--")
+    ax.plot(X_test, y_test + y_sigma, color="gray", ls="--")
+    ax.plot(X_test, y_pred, color="blue", alpha=0.5, label="Prediction intervals")
+    if title is not None:
+        ax.set_title(title)
+    ax.legend()
+```
+
+```python
+strategies = ["jackknife_plus", "jackknife_minmax", "cv_plus", "cv_minmax", "jackknife_plus_ab", "conformalized_quantile_regression"]
+n_figs = len(strategies)
+fig, axs = plt.subplots(3, 2, figsize=(9, 13))
+coords = [axs[0, 0], axs[0, 1], axs[1, 0], axs[1, 1], axs[2, 0], axs[2, 1]]
+for strategy, coord in zip(strategies, coords):
+    plot_1d_data(
+        X_train.ravel(),
+        y_train.ravel(),
+        X_test.ravel(),
+        y_mesh.ravel(),
+        (1.96*noise*X_test).ravel(),
+        y_pred[strategy].ravel(),
+        y_pis[strategy][:, 0, 0].ravel(),
+        y_pis[strategy][:, 1, 0].ravel(),
+        ax=coord,
+        title=strategy
+    )
+```
+
+We can observe that all of the strategies seem to have similar constant prediction intervals. 
+On the other hand, the quantile strategy offers a solution with prediction intervals
+that are smaller when x is small and that become larger as the noise in the data grows
```

Comment:
that adapts the prediction intervals to the local noise ?

Reply:
done